uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/FixNotMavenizedArtifactInfo.java

```diff
@@ -0,0 +1,90 @@
+package org.guvnor.m2repo.backend.server;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+
+import org.guvnor.common.services.project.model.GAV;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.ARTIFACT_ID;
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.GROUP_ID;
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.VERSION;
+
+/**
+ * This class aim to provide maven meta information for artifacts (.jar) that don't
```

Comment:
Thanks for the comment. I would have never figured out the intention of this just from the code.